src/pages/Landing/index.tsx

```diff
@@ -16,12 +16,22 @@ import {
 import { Caption, TableComposable, Tbody, Th, Thead, Tr } from '@patternfly/react-table';
 import { FormEvent, useState } from 'react';
 import { createUseStyles } from 'react-jss';
-import { useQuery } from 'react-query';
+import { useQuery, QueryClient, QueryClientProvider, } from 'react-query';
 import { choosableColors, Customer, getCustomers } from 'src/api/CustomerApi';
 import { ColoredTd } from 'src/components/ColoredTd';
 import Loader from 'src/components/Loader';
 import { useAppContext } from 'src/middleware';
+import ReactDOM from 'react-dom';
+import App from './App';
 
+const queryClient = new QueryClient();
+
+ReactDOM.render(
+  <QueryClientProvider client={queryClient}>
+    <App />
+  </QueryClientProvider>,
```

Comment:
So, this is a "Wrapper component" or "higher order component" (HOC). 
You need to add the queryClientProvider to wherever the top level of the app. 
In our apps case, we already have that done [here](https://github.com/content-services/React-typescript-demo/blob/main/src/index.tsx#L16C5-L16C47). 
Task 3 is asking you to use react-query to add the api calls to the modal component on submit.
IE call the api, with react query, on submit.